.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       uses: github/codeql-action/analyze@v1
 
     - name: Test
-      run: OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true ./gradlew test
+      run: OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true ./gradlew test -i
```

Comment:
Does this print info level logs on all tests or only failing tests?

Reply:
This prints logs for everything, which is the behavior when we were using maven